src/Engine/Component/RigidBody.cpp

```diff
@@ -103,7 +130,9 @@ namespace Component {
 
     glm::quat RigidBody::GetOrientation() const {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetOrientation' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)